packages/frontend/src/components/nft/NFTDetail.js

```diff
@@ -0,0 +1,163 @@
+        }
+
+        .desc {
+            margin-top: 48px;
+            margin-bottom: 56px;
+
+            font-weight: 500;
+            font-size: 16px;
+            line-height: 150%;
+            display: flex;
+            align-items: center;
+            color: #272729;
+        }
+    }
+
+    .owner {
+        p {
+            font-weight: 500;
+            font-size: 12px;
+            line-height: 150%;
+            display: flex;
+            align-items: center;
+            letter-spacing: 0.115em;
+            color: #A2A2A8;
+        }
+
+        .inner {
+            
+            display: flex;         
+            align-items: center;
+            word-break: break-all;
+            span {
+                font-weight: 500;
+                font-size: 16px;
+                line-height: 150%;
+                display: flex;
+                align-items: center;
+                color: #272729;
+            }
+        }
+    }
+
+    &&& {
+        .transfer-btn {
+            width: 100%;
+            margin-top: 56px;
+
+            svg {
+                margin-right: 10px;
+            }
+        }
+    }
+
+    .back-arrow-button {
+        position: absolute;
+        top: -12px;
+        left: -78px;
+    }
+
+`;
+
+const UserIcon = styled.div`
+    background-size: 21px;
+    flex: 0 0 40px;
+    height: 40px;
+    border-radius: 50%;
+    background-color: #f8f8f8;
+    text-align: center;
+    margin: 0 12px 0 0;
+    
+    svg {
+        width: 26px;
+        height: 26px;
+        margin: 7px;
+    }
+
+    @media (min-width: 940px) {
+        display: inline-block;
+    }
+`;
+
+export function NFTDetail({ nft, accountId, nearBalance, ownerId, history }) {
+    const [transferNftDetail, setTransferNftDetail] = useState();
+
+    return (
+        <StyledContainer className='medium centered'>
+          {
+            nft && 
+            <div className='container'>
+                <BackArrowButton
+                    onClick={() => history.goBack()}
+                    className='back-btn'
+                >
+                </BackArrowButton>
+
+                <img src={nft.metadata.mediaUrl} alt='NFT'/>
+                <h1 className="title">{nft.metadata.title}</h1>
+                <p className="desc">{nft.metadata.description}</p>
+
+                <div className='owner'>
+                    <p><Translate id='NFTDetail.owner'/></p>
+
+                    <div className='inner'>
+                        <UserIcon>
+                            <UserIconGrey color='#9a9a9a' />
+                        </UserIcon>
+                        <span>
+                            { ownerId }
+                        </span>
+                    </div>
+                </div>
+
+                {(ownerId === accountId) &&
+                  <FormButton 
+                    className='transfer-btn'
+                    color='gray-gray' 
+                    disabled={ownerId !== accountId || !nearBalance}
```

Comment:
1. :hocho: redundant check since this `FormButton` cannot be displayed if `ownerId !== accountId` is true due to the check on L140.
2. `nearBalance` will (almost?) never be falsy (absolutely 0), and even if it is a positive number, it may be too small to allow transfer of an NFT because it requires some gas be attached.  Should this be checking that the user has at least `NFT_TRANSFER_GAS + NFT_TRANSFER_DEPOSIT` available instead of checking this with a falsy check?